crates/ruff_python_ast/src/generated.rs

```diff
@@ -0,0 +1,8790 @@
+// This is a generated file. Don't modify it by hand!
+// Run `crates/ruff_python_ast/generate.py` to re-generate the file.
+
+/// See also [mod](https://docs.python.org/3/library/ast.html#ast.mod)
+#[derive(Clone, Debug, PartialEq, is_macro::Is)]
+pub enum Mod {
+    Module(crate::ModModule),
+    Expression(crate::ModExpression),
+}
+
+impl From<crate::ModModule> for Mod {
+    fn from(node: crate::ModModule) -> Self {
+        Self::Module(node)
+    }
+}
+
+impl From<crate::ModExpression> for Mod {
+    fn from(node: crate::ModExpression) -> Self {
+        Self::Expression(node)
+    }
+}
+
+impl ruff_text_size::Ranged for Mod {
+    fn range(&self) -> ruff_text_size::TextRange {
+        match self {
+            Self::Module(node) => node.range(),
+            Self::Expression(node) => node.range(),
+        }
+    }
+}
+
+/// See also [stmt](https://docs.python.org/3/library/ast.html#ast.stmt)
+#[derive(Clone, Debug, PartialEq, is_macro::Is)]
+pub enum Stmt {
+    #[is(name = "function_def_stmt")]
```

Comment:
Not for this PR but I think it would be nice if we generated the `is` implementations as well. The `is` macro doesn't always work well with ide's and macros are also more expensive to compile.